content/master/software/install.md

```diff
@@ -245,9 +245,8 @@ at the table below.
 {{< table caption="Feature flags" >}}
 | Status | Flag | Description |
 | --- | --- | --- |
-| Beta | `--enable-composition-revisions` |Enable support for CompositionRevisions |
+| Beta | `--enable-composition-revisions` | Enable support for CompositionRevisions. |
 | Beta | `--enable-composition-webhook-schema-validation` | Enable Composition validation using schemas. |
-| Beta | `--enable-deployment-runtime-configs` |Enable support for Deployment Runtime Configs. |
```

Comment:
Dropped by mistake?

Reply:
🤷‍♂️ this is what's in the docs today and was pulled in from the rebase. If this is wrong then we need to fix it in master.